eng/Versions.props

```diff
@@ -73,7 +73,7 @@
     <!-- CoreClr dependencies -->
     <MicrosoftNETCoreILAsmVersion>5.0.0-preview.8.20359.4</MicrosoftNETCoreILAsmVersion>
     <!-- Libraries dependencies -->
-    <StyleCopAnalyzersVersion>1.2.0-beta.205</StyleCopAnalyzersVersion>
+    <StyleCopAnalyzersVersion>1.2.0-beta.261</StyleCopAnalyzersVersion>
```

Comment:
Why does this need updating?

Reply:
Added a note in the commit message: https://github.com/dotnet/runtime/pull/45401/commits/43b9980b8089d8addfdc1e0173b664ed87cf2780. I used C# 9 new expression, where analyzer was was complaining. It is fixed in the newer version of analyzer package.

Reply:
In that case I would recommend removing the target-typed instances to avoid combining infrastructure updates with product code changes in the same commit. It makes reverting stuff easier.

Reply:
It looks like this was updated in the master branch. Reverted and rebased.